src/cordova.js

```diff
@@ -226,9 +226,7 @@ var cordova = {
             }
         } catch (err) {
             var msg = 'Error in ' + (isSuccess ? 'Success' : 'Error') + ' callbackId: ' + callbackId + ' : ' + err;
-            console && console.log && console.log(msg);
-            console && console.log && err.stack && console.log(err.stack);
-            cordova.fireWindowEvent('cordovacallbackerror', { 'message': msg });
+            cordova.fireWindowEvent('cordovacallbackerror', { 'message': msg, 'original': err });
```

Comment:
Would the property name `original` sound better if it is `error`, `stack`, or `errorStack`.

Once the property name is set and used, I am not sure if it would be changed it in the future if we find the name to be confusing.

People could build custom tooling that relays on this new field name and changing would break their tools.

Just want to make sure the meaning of the property carries over.

Reply:
I've had similar concerns. I think `error` would be a more meaningful property name.

Reply:
Good point.

This got me thinking.  What do you think of this instead?
```
err.message = msg;
corodva.fireWindowEvent(“cordovacallbackerror”, err);
```

This maintains backwards compatibility, but makes the error the top-level response.  

I think this makes a lot more sense because most people will probably think the object they receive in cordovacallbackerror will be an error.  (At least that is what I originally thought.)

Reply:
That might be even better. What do you think @erisu?

Reply:
I'm assuming in this context `err` is an `Error` object (since it comes from the `catch` block), assigning `err.message` would overwrite the original error message yes? If so, I don't think that's a good idea as that could be hiding important information for debugging. Or am I missing something?

Reply:
@breautek Yes, you are correct.  Though it is technically only prepending the error message with additional information. 

In the main comment stream, @raphinesse and I have been discussing this a bit.  (It is still up in the air).  If you would like to, please take a look and share your thoughts on the proposals.

Reply:
> Though it is technically only prepending the error message with additional information.

Ahh I see, that's where the `+ err` comes from. I think this is fine.

On the topic of using `error` vs `original`. +1 on `error`

Using `original` might make sense in the context of `cordovacallbackerror` but when you're debugging or something you just have a reference of the object (not necessary where it is coming from), the property name `original` isn't too meaningful of what the property is. I think using `error` as a property name is much more clear.